tests/integration/apis/clients/sponsored-brands-client.spec.ts

```diff
@@ -13,7 +29,151 @@ describe('SponsoredBrandsClient', () => {
 		});
 	});
 
+	beforeEach(() => {
+		const testName = expect.getState().currentTestName;
+		interceptorId = addInterceptor(sponsoredBrandsClient, testName);
+	});
+
+	afterEach(() => {
+		removeInterceptor(sponsoredBrandsClient, interceptorId);
+	});
+
 	it('should instantiate the api', () => {
 		expect(sponsoredBrandsClient).toBeDefined();
 	});
+
+	describe('#createCampaignsSb', () => {
+		// Test input gathered from this page -
+		// https://advertising.amazon.com/API/docs/en-us/bulksheets/sb/sb-examples/sb-create
+		// We need to override generated headers because Amazon docs are incorrect
+		// - returns 415 error with the generated headers
+		const options = { headers: { ['Content-Type']: 'application/json' } };
+
+		it('should create campaign of type SBCreateCampaignWithKeywords', async () => {
+			const creativeAsset: SBCreative = {
+				brandLogoAssetID: 'AWz-C_2InEZrJWCZIR5a',
+				brandName: 'Sample Creative Brand Name',
+				headline: 'Sample Creative Headline',
+			};
+
+			const landingPage: SBLandingPage = {
+				asins: ['B018UQ5AMS', 'B07PC7MHQ8', 'B07C1XC3GF'],
```

Comment:
Not sure these asins should be left in here.

Reply:
These are generic values, I copied from the API docs. Nothing client specific.